### ISSUE TITLE
Update intake tests

### ISSUE THREAD

**Issue by sjordan29** [xpublish-experiments/Catalog-To-Xpublish#13]
The reference links for the tests have been updated -- I already updated the STAC links, but the intake links and references also need to be updated now.

**Comment by sjordan29** [comment]
Updated all paths and tests are working locally, but failing on Github actions. 

```
=========================== short test summary info ============================
FAILED tests/test_1_intake_catalog_setup.py::test_catalog_classes - AssertionError: assert False
 +  where False = isinstance(<Intake catalog: test_intake_zarr_catalog>, <class 'intake.readers.entry.Catalog'>)
 +    where <Intake catalog: test_intake_zarr_catalog> = <catalog_to_xpublish.searchers.intake_search.IntakeCatalogSearch object at 0x7fd240036c10>.catalog_object
 +    and   <class 'intake.readers.entry.Catalog'> = intake.Catalog
FAILED tests/test_3_read_from_intake.py::test_read_from_intake - TypeError: ZarrArraySource.__init__() got an unexpected keyword argument 'consolidated'
ERROR tests/test_5_run_intake_server.py::test_docs - TypeError: ZarrArraySource.__init__() got an unexpected keyword argument 'consolidated'
ERROR tests/test_5_run_intake_server.py::test_json_endpoint - TypeError: ZarrArraySource.__init__() got an unexpected keyword argument 'consolidated'
ERROR tests/test_5_run_intake_server.py::test_yaml_endpoint - TypeError: ZarrArraySource.__init__() got an unexpected keyword argument 'consolidated'
ERROR tests/test_5_run_intake_server.py::test_catalog_endpoints - TypeError: ZarrArraySource.__init__() got an unexpected keyword argument 'consolidated'
ERROR tests/test_5_run_intake_server.py::test_datasets_endpoints - TypeError: ZarrArraySource.__init__() got an unexpected keyword argument 'consolidated'
ERROR tests/test_5_run_intake_server.py::test_zarr_endpoints - TypeError: ZarrArraySource.__init__() got an unexpected keyword argument 'consolidated'
============= 2 failed, 11 passed, 40 warnings, 6 errors in 14.77s =============
```

I think this may be because of the difference in intake in my local version vs the dev env created in gh actions (recent releases). Going to update the config specifying the intake version and see if that fixes it. Next thing to try after that might be specifying a zarr version.

**Comment by sjordan29** [comment]
Closed with #12 

Migrated intake issue to #14
